pkg/sink/kafka/sarama_factory.go

```diff
@@ -43,17 +46,32 @@ func NewSaramaFactory(
 }
 
 func (f *saramaFactory) AdminClient(ctx context.Context) (ClusterAdminClient, error) {
+	start := time.Now()
 	config, err := NewSaramaConfig(ctx, f.option)
+	duration := time.Since(start).Seconds()
+	if duration > 2 {
+		log.Warn("new sarama config cost too much time", zap.Any("duration", duration), zap.Stringer("changefeedID", f.changefeedID))
```

Comment:
please use debug log here

Reply:
I believe this should be a warning or info level log. A two-second threshold is quite large and this log could help identify potential issues.

Reply:
which condition may cost too much time on new the sarama config?

Reply:
Network partition.